src/entrypoint.sh

```diff
@@ -2,6 +2,7 @@
 
 set -o nounset
 set -o errexit
+# shellcheck disable=SC2039
```

Comment:
Don't we usually put in a comment explaining why a check is being disabled?

Reply:
Yes. I'll transfer commit comment to code: https://github.com/cisagov/certboto-docker/commit/8e9f90092dd5d98b8a8422e939d1c1e22ba0b9a0

Reply:
Comment added... good catch: https://github.com/cisagov/certboto-docker/pull/2/commits/db4220d1d138dfd0192a334ef7d6ee6f8812267c

Unfortunately I think my brain lets me get away with this when I write the commits with details.  Keep a look out!